.travis.yml

```diff
@@ -3,10 +3,10 @@ node_js:
   - "0.8"
   - "0.10"
   - "0.12"
-  - "1.0"
-  - "1.8"
-  - "2.0"
-  - "2.4"
+  - "iojs-1.0"
+  - "iojs-2.0"
```

Comment:
Please revert the changes to the way we do the version numbers. We purposely do not use "iojs" in the strings as well as include the top and bottom minor on purpose :)